OJP/OJP_Requests.xsd

```diff
@@ -299,4 +299,39 @@
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>========================================== Trip Monitoring Service (Subscription) ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:element name="OJPTripMonitoringSubscriptionRequest" type="OJPTripMonitoringSubscriptionRequestStructure" substitutionGroup="siri:AbstractFunctionalServiceSubscriptionRequest"/>
+	<xs:complexType name="OJPTripMonitoringSubscriptionRequestStructure">
+		<xs:complexContent>
+			<xs:extension base="AbstractOJPSubscriptionRequestStructure">
+				<xs:sequence>
+					<xs:element name="Trip" type="TripStructure"/>
+					<xs:element ref="OJPTripRequest" minOccurs="0"/>
+					<xs:element name="MonitoringParameters" type="TripMonitoringParamStructure" minOccurs="0"/>
+					<xs:group ref="AlertSettingsGroup"/>
+					<xs:element ref="siri:Extensions" minOccurs="0"/>
+				</xs:sequence>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:element name="OJPTerminateSubscriptionRequest" type="siri:TerminateSubscriptionRequestStructure">
+		<xs:annotation>
+			<xs:documentation>Request element for terminating subscriptions. To change parameters of a subscription, it is necessary to terminate and to re-subscribe. Successful termination is acknowledged by problem code TRIPMONITORING_SUBSCRIPTION_TERMINATED in a delivery.</xs:documentation>
+		</xs:annotation>
```

Comment:
According to the examples below, there is no *OJPTripMonitoringDelivery* in response to *siri:SubscriptionRequest* and *siri:TerminateSubscriptionRequest* - and therefore no possibility to directly return any of the *TRIPMONITORING_...* error codes. Is this a problem that needs to be solved or will the first reply (*siri:SubscriptionResponse*, e.g.) be followed by a second reply with a *OJPTripMonitoringDelivery*?

Reply:
Norman: No feedback yet. Investigating further
Matthias: Will check and if necessary error handling to be added.
Security considerations in short best practice paper (for all subscription related things from SIRI)
Andre: Currently we return a TripInformationDelivery on a Journey. We should be able to inform about a transfer leg. the question was, to how to do that.
Matthias: AlternativeTrip
Norman: Would solve it the way it was solved in TRIAS.

* Question to be prepared (Matthias, André)
* Norman asks TRIAS people (at Hafas)
* Adaption of the PR: (a) error handling (b) depending on result of TRIAS inquiry.
* Paper for security considerations added

Reply:
Norman: We had a research project in Shift2Rail. No production use for TripMonitoring. I can only describe how it was used. Only got notification. One has to add requests.  For more details TripInfoRequest is needed.
Currently no real world use, it might be not a good idea to add it. Perhaps postpone it to a later version. 

Postpone.